docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>(1) Why are they desirable? Can we add some explanation as why one might be interested in finding those patterns?</p><p> </p><p>(2) The code says <code>label="desired instances")</code> However, the figure's legend says "desired behaviour"</p><p> </p><p>(3) Right after this block, it might be useful to compute matrix profile, and discover motif and discord, and show that the discovered motifs/discords do not reveal the patterns we are looking for. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>1) add discription </p><p>2) fixed</p><p>3) add matrix profile section</p>